test_scripts/WebEngine/WebEngineProjection/028_OnExitApplication_unregister.lua

```diff
@@ -0,0 +1,40 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Processing of the OnExitApplication notification from HMI
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. Web app is registered
+
+-- Sequence:
+-- 1. HMI sends BC.OnExitApplication with 'reason' = "RESOURCE_CONSTRAINT" to SDL
+--  a. Web app is unregistered
+---------------------------------------------------------------------------------------------------
+--[[ General test configuration ]]
+config.defaultMobileAdapterType = "WS"
+
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Functions ]]
+local function onExitApp()
+  local appID = common.getHMIAppId()
+  common.getHMIConnection():ExpectNotification("BasicCommunication.OnAppUnregistered", {unexpectedDisconnect = false, appID = appID})
```

Comment:
@mvorobio This line too long. Please split it to 2 lines.
```
common.getHMIConnection():ExpectNotification("BasicCommunication.OnAppUnregistered", {unexpectedDisconnect = false, appID = appID})
```
=> 
```
common.getHMIConnection():ExpectNotification("BasicCommunication.OnAppUnregistered", 
    {unexpectedDisconnect = false, appID = appID})
```

Reply:
@aderiabin please check https://github.com/mvorobio/sdl_atf_test_scripts/pull/1/commits/9c41ad38677ff0bcac666a1a4b9ea393f4b8d49e